docs/core/mempool.md

```diff
@@ -46,3 +80,24 @@ order/nonce/sequence number, the application can reject transactions that are
 out of order. So if a node receives `tx3`, then `tx1`, it can reject `tx3` and then
 accept `tx1`. The sender can then retry sending `tx3`, which should probably be
 rejected until the node has seen `tx2`.
```

Comment:
Another approach would be to accept out of order transactions but reorder or exclude them from block proposals, during the execution or `PrepareProposal`